rfcs/text/pe/pe.yml

```diff
@@ -0,0 +1,257 @@
+      description: List of all imported functions
+      example: { "library_name" : "mscoree.dll", "imported_functions" : "GetFileVersionInfoSizeA" }
+
+    - name: sections
+      level: extended
+      description: >
+        Data about sections of compiled binary PE
+      type: nested
+      fields:
+        - name: chi2
+          level: extended
+          description: Chi-square probability distribution.
+          type: long
+          example: 3027194
+
+        - name: virtual_address
+          level: extended
+          description: Virtual address available to the file.
+          type: long
+          format: bytes
+          example: 8192
+
+        - name: entropy
+          level: extended
+          description: Measurement of entropy randomness in the file.
+          type: float
+          example: 6.24
+
+        - name: flags
+          level: extended
+          description: Section flags of the file.
+          type: keyword
+          example: rx
+
+        - name: name
+          level: extended
+          description: Section names of the file.
+          type: keyword
+          example: .text, .data
+
+        - name: raw_size
+          level: extended
+          description: Size of the section or the dize of the initialized data on disk.
+          type: long
+          format: bytes
+          example: 198144
+
+    - name: resources
+      level: extended
+      type: nested
+      description: >
+        If the PE contains resources, some info about them
+      fields:
+        - name: chi2
+          level: extended
+          description: Chi-square probability distribution.
+          type: long
+          example: -1
+
+        - name: filetype
+          level: extended
+          description: File type of the resources section.
+          type: keyword
+          example: Data
+
+        - name: entropy
+          level: extended
+          description: Measurement of entropy randomness in the resources section.
+          type: long
+          example: 0, 1
+
+        - name: sha256
+          level: extended
+          description: SHA256 hash of resources section.
+          type: keyword
+          example: e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
+
+        - name: language
+          level: extended
+          description: Language identification.
+          type: keyword
+          example: "CHINESE SIMPLIFIED"
+
+        - name: types
+          level: extended
+          type: keyword
+          short: List of resource types.
+          description: >
+            Digest of resource types. Key is resource type (as string) and value
+            is how many resources there are of that specific type (as integer)
+          example: { "RT_VERSION": 1, "RT_MANIFEST": 1 }
+
+    - name: exports
+      level: extended
+      type: keyword
+      description: >
+        List of symbols exported by PE
+      example: DllInstall, DllRegisterServer, DllUnregisterServer
```

Comment:
Is this field intended to be an array?

Reply:
Yeah, I think that makes sense.

So, would it be structured like this?
```
example: { "DllInstall", "DllRegisterServer", "DllUnregisterServer" }
```

Reply:
The example value can be quoted and escaped to avoid unexpected YAML interpretation.

Listing `array` under `normalize` declares that a field's value is always expected to be an array, even if there's only a single value.

```yaml
    - name: exports
      level: extended
      type: keyword
      description: >
        List of symbols exported by PE
      example: "[\"DllInstall\", \"DllRegisterServer\", \"DllUnregisterServer\"]"
      normalize:
        - array
```

Reply:
Yep, I'll update that.